webapp/src/ts/components/filters/multi-dropdown-filter/mullti-dropdown-filter.component.ts

```diff
@@ -82,6 +88,18 @@ export class MultiDropdownFilterComponent implements AbstractFilter {
 
   clear(apply=true) {
     this.selected.clear();
-    apply && this.apply();
+    if (apply) {
+      return this.apply();
+    }
+
+    this.filterLabel = this.getLabel();
   }
 }
+
+export class MultiDropdownFilterComponentStub {
```

Comment:
> I would have MultiDropdownFilter as a separate file
I think that just bloats the directory view. MultiDropdownFilter is very much an annex.

Reply:
I'm changing the name to `MultiDropdownFilter`, but I think that needing to call `super` for nothing in the actual component is really unnecessary and might be even more confusing.

Reply:
> I want to also point out that exporting multiple connected things from a single file is a common occurrence
yeah, but if we ever have more flavours of the dropdown, then makes sense to extract in a separate file :) 
> but I think that needing to call super for nothing in the actual component
Since MultiDropdownFilter isn't defining the function's body and it's being done by the component, that's why.  

Thanks for taking a look at this, this is fine for now.